helper_methods.rb

```diff
@@ -0,0 +1,127 @@
+module HelperMethods
+
+  def register_user
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+    @login = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys(@login)
+    @driver.find_element(:id, 'user_lastname').send_keys(@login)
+    @driver.find_element(:id, 'user_mail').send_keys(@login + '@q.com')
+    @driver.find_element(:name, 'commit').click
+  end
+
+  def register_user1
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+    @login1 = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login1)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys(@login1)
+    @driver.find_element(:id, 'user_lastname').send_keys(@login1)
+    @driver.find_element(:id, 'user_mail').send_keys(@login1 + '@q.com')
+    @driver.find_element(:name, 'commit').click
+  end
+
+  def login_user
+    register_user
+    @driver.find_element(:class, 'logout').click
+    @driver.find_element(:class, 'login').click
+    @driver.find_element(:id, 'username').send_keys @login
+    @driver.find_element(:id, 'password').send_keys @password
+    @driver.find_element(:name, 'login').click
+  end
+
+  def change_password
+    register_user
+    @new_password = @password + '1'
+    @driver.find_element(:class, "my-account").click
+    @driver.find_element(:xpath, ".//*[@id='content']//a[@class='icon icon-passwd']").click
+    @driver.find_element(:id, "password").send_keys(@password)
+    @driver.find_element(:id, "new_password").send_keys(@new_password)
+    @driver.find_element(:id, "new_password_confirmation").send_keys(@new_password)
+    @driver.find_element(:xpath, "//*[@name='commit']").click
+  end
+
+  def create_project
+    register_user
```

Comment:
the `register_user` method shouldn't be part of `create_project` method
so when we need to create project we will call `register_user` first and then `create_project`.
Because these are completely different logical actions